webtoolbar.py

```diff
@@ -209,6 +209,28 @@ def __key_press_event_cb(self, entry, event):
         selection = self._search_view.get_selection()
         model, selected = selection.get_selected()
 
+        if event.get_state() & Gdk.ModifierType.CONTROL_MASK:
+            if keyname == 'Return':
+                m = re.search('\.', self.props.text)
```

Comment:
You can use `m = '.' in self.props.text` and then you don't waste compute power with regex.